docs/usage/commands-handling.md

```diff
@@ -56,6 +56,7 @@ options["result_function"](
 )
 ```
 
+(reporting-failure)=
```

Comment:
First time i see these anchors. I typically just use the title but replacing the spaces with dashes